packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -218,7 +218,11 @@ class Auth {
                 siteId: config.siteId
             },
             throwOnBadResponse: true,
-            fetchOptions: config.fetchOptions
+            // We need to set credentials to 'include' to allow cookies to be set. This is required as SLAS calls return a dwsid cookie for hybrid sites.
```

Comment:
```suggestion
            // We need to set credentials to 'same-origin' to allow cookies to be set. This is required as SLAS calls return a dwsid cookie for hybrid sites.
```

It fees like this comment doesn't explain the why. I think it's important to keep a record that this is for basket server affinity and sending dwsid cookie is the way to achieve that.

Reply:
Agree

Reply:
Added a comment explaining the details